### ISSUE TITLE
NTR: Phosphomannose isomerase deficiency in leukocytes, fibroblasts, or liver (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8136]
New term request Phosphomannose isomerase deficiency in leukocytes, fibroblasts, or liver (segal_040409164830):   UMLS: 1865146

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000782
- label: Reduced tissue phosphomannose isomerase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:9525984
- def: Activity of the mitochondrial enzyme phosphomannose isomerase (PMI; EC 5.3.1.8) in tissues below the lower limit of normal. The activity can be measured in multiple tissues including leukocytes, fibroblasts, and liver.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8136
- synonyms:
